packages/contracts-bedrock/deploy-config/getting-started.json

```diff
@@ -29,10 +29,18 @@
   "finalizationPeriodSeconds": 12,
 
   "proxyAdminOwner": "ADMIN",
+
```

Comment:
nit: think this newline (between `proxyAdminOwner` and `baseFeeVaultRecipient` was added by mistake
```suggestion
```